pandas/plotting/_matplotlib/converter.py

```diff
@@ -426,12 +429,9 @@ def __call__(self):
         ed = _from_ordinal(dates.date2num(dmax))
         all_dates = date_range(start=st, end=ed, freq=freq, tz=tz).astype(object)
 
-        try:
-            if len(all_dates) > 0:
-                locs = self.raise_if_exceeds(dates.date2num(all_dates))
-                return locs
-        except Exception:  # pragma: no cover
-            pass
```

Comment:
This seems to be a change in behaviour? Is this tested? Or do you think this could never raise?

Reply:
the `len` check I dont think can raise.  the `self.raise_if_exceeds` seems like the kind of thing that could raise, but I think thats a matplotlib method so I'll have to dig into that to find a failing case.